src/ge-mri/rds.jl

```diff
@@ -0,0 +1,69 @@
+export loadrds
+
+"""
+    dat = loadrds(fid::IOStream, frame::Int, frsize::Int, ncoil::Int; 
+                  ptsize::Int = 2)
+
+Load one shot (frame) from MRI data file created with GE's Raw Data Server.
+
+in
+- `fid::IOStream`
+  File handle
+- `frame::Int`
+  Shot/frame number
+- `frsize::Int`
+  Number of data points in each readout (per coil)
+- `ncoil::Int`
+  Number of receive coils
+
+option
+- `ptsize::Int`
+  Either 2 (real/complex value stored as Int16; default)
+  or 4 (Int32)
+  
+out
+- `dat::Array{Complex{Int16/Int32}}` `[frsize, ncoil]`
+
+"""
+function loadrds(fid::IOStream, frame::Int, frsize::Int, ncoil::Int; 
+    ptsize::Int = 2,
+)
+
+    if ptsize == 2
+        d = Array{Complex{Int16}}(undef, frsize*ncoil) # one readout
+    else
+        d = Array{Complex{Int32}}(undef, frsize*ncoil)
+    end
+
+    seek(fid, 2*ptsize*(frame-1)*frsize*ncoil)
+
+    read!(fid, d)
```

Comment:
```suggestion
    read!(fid, vec(d))
```
this uses a 1D "view" of d (like a pointer) so avoids duplicate allocations

Reply:
Neat, thanks!